### ISSUE TITLE
Tools seen as malware from python3-impacket dependency

### ISSUE THREAD

**Issue by raschidb** [grindsa/acme2certifier#181]
Hi @grindsa 
One more time thank you for your work
I have installed acme2certifier, and our cisrt team has open an issue for malware tools installed.
In /usr/bin/ : mimikatz ; mssqlcient ; sniff-3.9, etc...
After investigating we found that they come from the dependency : python3-impacket
Is there a way to remove this a package ? or is there another one we can use without those dangerous tools on the server ?
amce2certifier v 0.35 on redhat9

Thank you
Rachid.

**Comment by grindsa** [comment]
Hi @raschidb 

python3-imacket is only be used in case you are using the [mswcce ca-handler](https://github.com/grindsa/acme2certifier/blob/master/docs/mswcce.md). If you are not using this handler you can remove the package. Can you pls name the scanner your cisrt team is using and provide the full output? I can look for a workaround (like creating a package without the tools) then.

**Comment by raschidb** [comment]
Yes, the handler is mswcce_ca_hanlder.
The scan has been performed by cyber reason  https://www.cybereason.com/
Unfortunately we cannot provide the full output, but they said every security scanner should warn about this library.
If the purpose is for RPC they advise to use tinyRPC module instead.

Thank you
Rachid

**Comment by raschidb** [comment]
ok i will see tomorow if a can grab the full list.
But i think it's all the file installed in /usr/bin/ they all look like hacking tool:
Thank you

**Comment by raschidb** [comment]
Hi @grindsa 
Please find the file list below :

_time  file_path  file_hash  signature
2024-09-04 03:03:47.794  /usr/bin/mimikatz-3.9  a8b0605e0932bc003f70a16682964da4  Application.Generic.3674855
2024-09-04 03:03:47.800  /usr/bin/mqtt_check-3.9  44c2d647599437d673590b7e9480c6ff  Trojan.Generic.35724109
2024-09-04 03:03:47.805  /usr/bin/mssqlclient-3.9  3cd59f3b8719e55fe12a04d78a8d7020  Trojan.Generic.35724146
2024-09-04 03:03:47.820  /usr/bin/netview-3.9  b59a56b3b1c35df41c2102913b96dc4c  Trojan.Generic.35724173
2024-09-04 03:03:47.837  /usr/bin/ntfs-read-3.9  ac70cfe1312dfc7e54d40161c65819c7  Trojan.Generic.35724174
2024-09-04 03:03:47.846  /usr/bin/ntlmrelayx-3.9  f3c026502592d4418628772b41dad67c  Application.Generic.3674857
2024-09-04 03:03:47.869  /usr/bin/psexec-3.9  1e1e7dec2c6b885847b3a8e6ebb49ec9  Application.Generic.3674858
2024-09-04 03:03:47.887  /usr/bin/raiseChild-3.9  6885d8c9cabb96f596788cd21e7eedbe  Application.Generic.3674850
2024-09-04 03:03:47.916  /usr/bin/reg-3.9  691811c19c58bc900b61aeb20f4b0390  Trojan.Generic.35724121
2024-09-04 03:03:47.931  /usr/bin/rpcdump-3.9  b902ff60811ff396371798b014e4b13f  Trojan.Generic.35724175
2024-09-04 03:03:47.950  /usr/bin/sambaPipe-3.9  435328d2a122fd7d5df57fdc6c8d2e4a  Trojan.Generic.35724108
2024-09-04 03:03:47.956  /usr/bin/samrdump-3.9  1e06d0461ab1047452f07cbe0d4555e2  Trojan.Generic.35724189
2024-09-04 03:03:47.963  /usr/bin/secretsdump-3.9  711cdc3c739abe2df55ad517e71962b0  Application.Generic.3674851
2024-09-04 03:03:47.969  /usr/bin/services-3.9  559bb31c097f460a8697c9edc0726f68  Trojan.Generic.35724115
2024-09-04 03:03:47.976  /usr/bin/smbclient-3.9  99e3f09367df958939fe7441695dc5bd  Trojan.Generic.35724168
2024-09-04 03:03:47.985  /usr/bin/smbexec-3.9  c4140e9730bad8d498c372290344539d  Application.Generic.3674860
2024-09-04 03:03:47.992  /usr/bin/smbpasswd-3.9  23bc8d6ce52275ba65337ab1f25cf363  Trojan.Generic.35724191
2024-09-04 03:03:48.006  /usr/bin/smbrelayx-3.9  9b028cce9ed27e66ee88906a862d54ef  Trojan.Generic.35724169
2024-09-04 03:03:48.012  /usr/bin/smbserver-3.9  349dea271f401565a49f2a993a205502  Trojan.Generic.35724194
2024-09-04 03:03:48.039  /usr/bin/ticketer-3.9  1a652dc245711a42c5ce255b8473329b  Trojan.Generic.35724187
2024-09-04 03:03:48.046  /usr/bin/wmiexec-3.9  e768fd7ddf4670f1918da630074002d8  Application.Generic.3674862
2024-09-04 03:03:48.052  /usr/bin/wmipersist-3.9  516321a37037197e8ade34e7df65d672  Trojan.Generic.35724113
2024-09-04 03:03:48.057  /usr/bin/wmiquery-3.9  8fefdddd19c6739c605e7e972ec42121  Trojan.Generic.35724163
2024-09-04 03:03:48.142  /usr/bin/GetADUsers-3.9  4cedadf92713122f885b535263d129df  Trojan.Generic.35724112
2024-09-04 03:03:48.152  /usr/bin/GetNPUsers-3.9  c1d9eb485b657c267cc8e86fe0efe98a  Trojan.Generic.35724178
2024-09-04 03:03:48.161  /usr/bin/GetUserSPNs-3.9  416778e0cabbae9a5413086c3d3aed65  Trojan.Generic.35724107
2024-09-04 03:03:48.165  /usr/bin/addcomputer-3.9  eae95cc8a6a1d9b7c3af1af5ca3767a0  Trojan.Generic.35724182
2024-09-04 03:03:48.174  /usr/bin/atexec-3.9  42b33bd69a054e3bce5462eadede569e  Application.Generic.3674848
2024-09-04 03:03:48.183  /usr/bin/dcomexec-3.9  5bbc84ad9527d9625aad183124191091  Application.Generic.3674849
2024-09-04 03:03:48.191  /usr/bin/dpapi-3.9  cdf747b2af0ae1c2bee99c5df17d7929  Trojan.Generic.35724180
2024-09-04 03:03:48.197  /usr/bin/esentutl-3.9  486c877b15c25b5461a72dedc75a14b2  Trojan.Generic.35724111
2024-09-04 03:03:48.214  /usr/bin/findDelegation-3.9  6f3aeee3591e68e18a4ca6bd1176ebe9  Trojan.Generic.35724125
2024-09-04 03:03:48.221  /usr/bin/getArch-3.9  6ca7e0543e421298b02631c244d619c9  Trojan.Generic.35724123
2024-09-04 03:03:48.229  /usr/bin/getPac-3.9  e4b0fa02735e05c135bf5e90c4560960  Trojan.Generic.35724149
2024-09-04 03:03:48.239  /usr/bin/getST-3.9  5383873ff746ed09f7f2a4b68d939856  Trojan.Generic.35724114
2024-09-04 03:03:48.246  /usr/bin/getTGT-3.9  ba334f4abfd83b4fd8a0897928abe392  Application.Generic.3674856
2024-09-04 03:03:48.256  /usr/bin/goldenPac-3.9  1fdbf064cd3a253231dabad98e86f53a  Application.Generic.3674859
2024-09-04 03:03:48.268  /usr/bin/keylistattack-3.9  70baf77860e946db6e8cd1efc6e8b515  Trojan.Generic.35724127
2024-09-04 03:03:48.290  /usr/bin/machine_role-3.9  9d548dd7c78057f28987c700edfaa7ef  Trojan.Generic.35724170
 

Thank you
Rachid

**Comment by grindsa** [comment]
thx, thats enough. I created a stripped down version of python3-impacket without the tools reported by the scanner. 

[python3-impacket-0.11.0-2grindsa.el9.noarch.rpm.gz](https://github.com/user-attachments/files/16897986/python3-impacket-0.11.0-2grindsa.el9.noarch.rpm.gz)

I did some quick tests and could not find issues.  The new rpm can installed via `yum localinstall ...`. Feel free to give it a try and let me know if it works for you and if your cisrt guys look happier now.
